src/main/scala/supporters/DefaultMapsContributor.scala

```diff
@@ -8,8 +8,10 @@ package viper.silicon.supporters
 
 import scala.reflect.{ClassTag, classTag}
 import viper.silicon.Config
+import viper.silicon.common.collections.immutable.InsertionOrderedSet
```

Comment:
Since nothing else changed in the file I'd assume that the added imports aren't needed